nntrainer/include/tensor.h

```diff
@@ -474,6 +486,7 @@ class Tensor {
   static constexpr float min_limits = std::numeric_limits<float>::min();
   static constexpr float max_limits = std::numeric_limits<float>::max();
   template <typename T> void setDist(T dist);
+  float epsilon = 1e-5;
```

Comment:
Maybe we could get this out to a `constant` used in `nntrainer` later? There are many operations using custom epsilon...

```suggestion
  static constexpr float epsilon = 1e-5;
```

Reply:
Updated to `static constexpr`. Later, I have to update to dynamic epsilon where we just look at mantissa number of bits.
This will move to a different header then.
@zhoonit @jijoongmoon Should I make tensor depend on gtest and use its float comparison directly, or implement a similar one for future?